providers/aws/resources/aws_s3.go

```diff
@@ -133,6 +158,9 @@ func initAwsS3Bucket(runtime *plugin.Runtime, args map[string]*llx.RawData) (map
 	var arn string
 	if args["arn"] != nil {
 		arn = args["arn"].Value.(string)
+		if !strings.HasPrefix(arn, "arn:aws:s3:") {
+			return nil, nil, errors.Newf("not a valid bucket ARN '%s'", arn)
```

Comment:
During testing I detected that, if you are NOT in discovery and, you try to access `aws.s3.bucket` then we try to "create a bucket resource" of the AWS account, which is incorrect.

Now, we will exit with helpful error message:
```
cnquery> aws.s3.bucket
not a valid bucket ARN 'arn:aws:sts::123456789012'
aws.s3.bucket: no data available
```